app/assets/javascripts/app/views/location_stream.js

```diff
@@ -1,6 +1,6 @@
 // @license magnet:?xt=urn:btih:0b31508aeb0634b347b8270c7bee4d411b5d4109&dn=agpl-3.0.txt AGPL-v3-or-Later
 
-app.views.LocationStream = app.views.Content.extend({
+app.views.LocationStream = app.views.Base.extend({
```

Comment:
@denschub Looks like there is none. The only thing we need in the location template is the location object which we can get directly from the model. Thus even with the following changes the code seems to be working fine:

```diff
diff --git a/app/assets/javascripts/app/views/content_view.js b/app/assets/javascripts/app/views/content_view.js
index 485fca01c..7027ea97c 100644
--- a/app/assets/javascripts/app/views/content_view.js
+++ b/app/assets/javascripts/app/views/content_view.js
@@ -10,7 +10,6 @@ app.views.Content = app.views.Base.extend({
       text : app.helpers.textFormatter(this.model.get("text"), this.model.get("mentioned_people")),
       largePhoto : this.largePhoto(),
       smallPhotos : this.smallPhotos(),
-      location: this.location(),
       isReshare : this.model.get("post_type") === "Reshare"
     });
   },
@@ -38,11 +37,6 @@ app.views.Content = app.views.Base.extend({
     $(evt.currentTarget).hide();
   },
 
-  location: function(){
-    var location = this.model.get("location")? this.model.get("location") : "";
-    return location;
-  },
-
   collapseOversized : function() {
     var collHeight = 200
       , elem = this.$(".collapsible")
diff --git a/app/assets/javascripts/app/views/location_stream.js b/app/assets/javascripts/app/views/location_stream.js
index 925ee1f2f..8b3238c30 100644
--- a/app/assets/javascripts/app/views/location_stream.js
+++ b/app/assets/javascripts/app/views/location_stream.js
@@ -1,6 +1,6 @@
 // @license magnet:?xt=urn:btih:0b31508aeb0634b347b8270c7bee4d411b5d4109&dn=agpl-3.0.txt AGPL-v3-or-Later
 
-app.views.LocationStream = app.views.Content.extend({
+app.views.LocationStream = app.views.Base.extend({
   events: {
     "click .near-from": "toggleMap"
   },
```